src/libsync/propagateupload.h

```diff
@@ -124,6 +124,45 @@ private slots:
 };
 
 /**
+ * @brief The MultipartJob class
+ * @ingroup libsync
+ */
+class MultipartJob : public AbstractNetworkJob {
```

Comment:
Not convinced, the goal with bundling is to bundle all requests in Bunlemetadata JSON Object